org-ql-view.el

```diff
@@ -299,7 +299,10 @@ update search arguments."
     (goto-char (point-min))
     (or (when (search-forward current-line nil t)
           (beginning-of-line))
-        (goto-char old-pos))))
+        (goto-char old-pos))
+        (message "org-ql: view refreshed%s"
+                 (when org-ql-view-title
+                   (concat ": " (substring-no-properties org-ql-view-title))))))
```

Comment:
Is it necessary to put the view title in the message?  Won't the user be looking at the view buffer when he refreshes it?

Reply:
Fair point, removed.